languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -9065,19 +9065,35 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
         </rulegroup>
         <rulegroup id="JE_DESTO" name="je/desto">
             <rule>
+                <antipattern>
+                    <token>je</token>
+                    <token regexp="yes">.+mal</token>
+                </antipattern>
+                <antipattern>
+                    <token>je</token>
+                    <token postag="ADJ:PRD:KOM|PA2:PRD:KOM:.*|ADV:.*" postag_regexp="yes" skip="-1"/>
+                    <token min="0">je</token>
+                    <token min="0" postag="ADJ:PRD:KOM|PA2:PRD:KOM:.*|ADV:.*" postag_regexp="yes" skip="-1"/>
+                    <token min="0">je</token>
+                    <token min="0" postag="ADJ:PRD:KOM|PA2:PRD:KOM:.*|ADV:.*" postag_regexp="yes" skip="-1"/>
+                    <token min="0">je</token>
+                    <token min="0" postag="ADJ:PRD:KOM|PA2:PRD:KOM:.*|ADV:.*" postag_regexp="yes" skip="-1"/>
+                    <token regexp="yes">desto|umso</token>
+                </antipattern>
                 <pattern>
                     <token>je</token>
-                    <token postag="ADJ:PRD:KOM" skip="-1"><exception regexp="yes" scope="next">bzw|und</exception></token>
+                    <token postag="ADJ:PRD:KOM|PA2:PRD:KOM:.*|ADV:.*" postag_regexp="yes" skip="-1"/>
```

Comment:
Nachdem die Regel schon aktiv ist, würde ich sie lieber vorsichtig testen:

- Deine Regel `JE_DESTO_TEST` nennen und höher priorisieren als `JE_DESTO`
- Diff checken, ob erstere korrekt funktioniert und ob keine True Positives abhanden kommen
- wenn alles okay, die alte durch die neue ersetzen (und eintrag in German.java wieder entfernen)